package/skelcd-control-openSUSE.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Jun  1 13:21:39 CEST 2017 - schubi@suse.de
+
+- control.cml: "custom" role has an own pattern selection screen
+  mask while the installation workflow. The default values can be
```

Comment:
It is hard to understand, I'd suggest something like this for the whole changelog:

*The "custom" role displays manual the pattern selection dialog in the installation workflow. The default pre-selected patterns can be configured in the "control.xml" file in the "system_role" section.*